Detections/CarbonBlack/KnownMalwareDetected.yaml

```diff
@@ -0,0 +1,26 @@
+id: 9f86885f-f31f-4e66-a39d-352771ee789e
+name: Known Malware Detected
+description: |
+  'This creates an incident when a known Malware is detected on a endpoint managed by a Carbon Black.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: CarbonBlackEvents_CL
```

Comment:
Same as previous for the connectorId and dataTypes.

Reply:
corrections have been made and pushed.